roles/cloud-bootstrap/tasks/create-users-projects.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create bonnyci project
+  os_project:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci-admin group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci-admin
+
+- name: Add all users to bonnyci group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci
+    user: "{{ item.username }}"
+  with_items: users
+
+- name: Add all bonnyci group members to bonnyci project
+  os_user_role:
+    cloud: "{{ cloud }}"
+    role: member
```

Comment:
this role should be `_member_`, but also should be a variable.

In fact, the existing role `users-projects` did this bit. Your commit sort of leaves that role an orphan. Could that code instead be used in this role?